app/views/devise/ichain_sessions/new.html.haml

```diff
@@ -19,6 +21,6 @@
       .form-group
         = button_tag(type: 'submit', class: "btn btn-success pull-right") do
           = content_tag(:strong, 'Sign In')
-  
+
     - if devise_mapping.ichain_registerable?
-      = link_to "Sign up", new_ichain_registration_path(resource_name)
+      = link_to "Sign up", 'https://idp-portal.suse.com'
```

Comment:
This is a function of the login proxy. It also takes care of redirecting etc. We shouldn't change this.